dev/tools/make-changelog.sh

```diff
@@ -17,10 +17,15 @@ where="$where/$PR-$(git rev-parse --abbrev-ref HEAD).rst"
 # if necessary but doesn't hurt)
 printf '%s bla bla (`#%s <https://github.com/coq/coq/pull/%s>`_, by %s).' - "$PR" "$PR" "$(git config user.name)" > "$where"
 
+printf "Name of created changelog file:\n"
+printf "$where\n"
+
 giteditor=$(git config core.editor)
 if [ "$giteditor" ]; then
     $giteditor "$where"
 elif [ "$EDITOR" ]; then
     $EDITOR "$where"
-else echo "$where"
+else
+    printf "The file has now to be edited"
```

Comment:
How about "describe the changes in the above file" or some such thing?

Reply:
OK. I pushed your proposition.